instruments/yokogawa/yokogawa6370.py

```diff
@@ -183,13 +183,13 @@ def data(self):
         """
         Function to query the active Trace data of the OSA.
         """
-        return self.channel[self.active_trace].data
+        return self.channel[self.active_trace].data()
 
     def wavelength(self):
         """
         Query the wavelength axis of the active trace.
         """
-        return self.channel[self.active_trace].wavelength
+        return self.channel[self.active_trace].wavelength()
```

Comment:
I think there is a TODO brewing here. I'd say out of scope for this PR, but we should think about if we want to make `wavelength` a property

Reply:
These two methods for `data` and `wavelength` are simply a shortcut to access the currently active channel. The channels themselves in fact also have the additional arguments for `bin_format`:

```python
def data(self, bin_format=True):
```

For oscilloscopes for example, getting data is also always defined as methods to allow for a `bin_format` argument. If we want to turn it into a method there is the question on how to implement the `bin_format`, maybe with a separate property on the data transfer format? If it is left as a method, it would be good to pass the `bin_format` through in these methods. What do you think? That latter could easily be included here.

Reply:
I think this is something we will have to think about by taking a look at the manual and seeing how we might expect the user to use the wavelength data here. I'm not super familiar with the instrument so I would need to check that first.

This is def a later-thing though, and I'm okay with how it is right now!